### PR TITLE
Grid Simplified

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: components/Grid Simplified/grid.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+.txt{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.flex-txt{
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
```
Comment (aamrapaliwandhre) [inline components/Grid Simplified/grid.css]: remove this comment

File: components/Grid Simplified/grid.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<body>
+    <div class="content">
+        <section>
+        <h1>Grid Simplifed</h1> 
+            <p> Use grid when you want sections in view. Check below some examples of side by side cards.</p> 
+         
+          <h1>Example of Grid 50:50 Layout</h1>
+            <div class="grid-box">
+                <div class="grid-content">
+                    <img src="/images/landging -page 5.png" alt="">
```
Comment (aamrapaliwandhre) [inline components/Grid Simplified/grid.html]: alt is empty...give it name...

Comment (iamharnad) [inline components/Grid Simplified/grid.html]: fill the alt attribute for the image tag. its for better coding practise

File: components/Grid Simplified/grid.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+              style="width: 770px; height: 346px; border:0; transform: scale(1); overflow:hidden;"
+              sandbox="allow-scripts allow-same-origin">
+            </iframe> </div>
+          <h1>Example of Grid 70:30 Layout</h1>
+            <div class="grid-box-content">
+                <div class="grid-content">
+                    <img src="/images/landging -page 5.png" alt="">
```
Comment (iamharnad) [inline components/Grid Simplified/grid.html]: Here too

Comment (aamrapaliwandhre) [inline components/Grid Simplified/grid.html]: don't leave alt empty...

File: components/Grid Simplified/grid.css (hunk 1 above)
Comment (Prabhav-Sharma) [inline components/Grid Simplified/grid.css]: Remove comment

File: components/Grid Simplified/grid.html (hunk 3 above)
Comment (iamharnad) [inline components/Grid Simplified/grid.html]: Here as well

**Review by aamrapaliwandhre — APPROVED** [review]
other than small suggested changes code looks fine to me....

**Review by iamharnad — CHANGES_REQUESTED** [review]
fill the alt attribute for the image tag. its for better coding practise. Otherwise everything is good

File: components/Grid Simplified/grid.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+}
+.grid-content{
+    margin: 2rem;
+    border: 1px solid #797572;
+}
+.grid-box{
+    width: 100%;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    border: 4px solid #6b6a6a;
```
Comment (mehulsatardekar) [inline components/Grid Simplified/grid.css:28]: @jeetbhalani15 I see that this color is being used repeatedly so use var for that

File: components/Grid Simplified/grid.html (hunk 3 above)
Comment (jeetbhalani15) [reply components/Grid Simplified/grid.html]: done

File: components/Grid Simplified/grid.css (hunk 1 above)
Comment (jeetbhalani15) [reply components/Grid Simplified/grid.css]: sure

File: components/Grid Simplified/grid.html (hunk 3 above)
Comment (mehulsatardekar) [inline components/Grid Simplified/grid.html]: you have empty alt tag inside img tag so provide some text to it

**Review by mehulsatardekar — APPROVED** [review]
@jeetbhalani15 good to merge the branch with the changes

**Review by shivamSspirit — COMMENTED** [review]
good look to merge

**Pull request merged**